lib/open-search-client.test.ts

```diff
@@ -42,70 +54,40 @@ describe('open-search-client', () => {
 
   describe('openScroll', () => {
     test('should return scrolled search results', async () => {
-      nock('https://open-search.example.com')
-        .post('/documents/_search')
-        .reply(200, {
-          _scroll_id: 'mockScrollId1',
-          hits: {
-            hits: [], // We don't actually care about faking hits.
-          },
-        });
-      const result = await osClient.openScroll(
-        'https://open-search.example.com',
-        'documents',
-        { size: 2 }
-      );
+      const response = await osClient.openScroll(esUrl, index, { size: 2 });
 
-      expect(result).toEqual({
-        _scroll_id: 'mockScrollId1',
-        hits: {
-          hits: [], // We don't actually care about faking hits.
-        },
-      });
+      expect(response)
+        .toMatchZodSchema(openSearchScrollDocumentsResponseSchema);
```

Comment:
Unless I'm missing it we haven't actually put anything in the index yet, right? Should we do that first since the documents response schema specifically validates the `_source` field of the scroll response?

Reply:
The previous implementation of this test was returning an empty array for `hits.hits`, so I figured that returning no tests in this case was fine as well. Should we be expanding the test here?

Reply:
Yeah that's fine for now I guess.